vector/src/main/java/im/vector/app/features/home/room/detail/TimelineFragment.kt

```diff
@@ -1475,8 +1475,9 @@ class TimelineFragment @Inject constructor(
                     attachmentTypeSelector.setAttachmentVisibility(
                             AttachmentTypeSelectorView.Type.LOCATION,
                             vectorPreferences.isLocationSharingEnabled())
-                    attachmentTypeSelector.setAttachmentVisibility(
-                            AttachmentTypeSelectorView.Type.POLL, !isThreadTimeLine())
+                    // Tchap: Disable Polls
+//                    attachmentTypeSelector.setAttachmentVisibility(
+//                            AttachmentTypeSelectorView.Type.POLL, !isThreadTimeLine())
```

Comment:
I think you can revert the xml change and only pass false in this method?

Reply:
nice, it's working. Thank you